annotations/views.py

```diff
@@ -364,6 +366,62 @@ def collection_texts(request, collectionid):
     return HttpResponse(template.render(context))
 
 
+class RecentActivity(models.Model):
```

Comment:
@kardanitin @apchandak24 Take a look at [collections](https://docs.python.org/2/library/collections.html). We don't want to keep things as simple as possible, so definitely take advantage of existing classes. Even a ``dict`` would be enough to count beans, but something like a ``Counter`` or ``defaultdict`` (from ``collections``) might be better.

But again, this may or may not be necessary after using ``annotate``, as discussed.